packages/headless/src/features/facets/automatic-facets/automatic-facets-actions-loader.ts

```diff
@@ -0,0 +1,33 @@
+import {PayloadAction} from '@reduxjs/toolkit';
+import {SearchEngine} from '../../../app/search-engine/search-engine';
+import {setDesiredCount} from './automatic-facets-actions';
+import {automaticFacetsReducer as automaticFacets} from './automatic-facets-slice';
+
+/**
+ * The action creators for managing automatic facets.
+ */
+export interface AutomaticFacetsActionCreators {
+  /**
+   * Sets the desired count of automatic facets.
+   *
+   * @param desiredCount - Desired count of automatic facets.
+   * @returns A dispatchable action.
+   */
+  setDesiredCount(desiredCount: number): PayloadAction<number>;
+}
+
+/**
+ * Loads the automatic facets actions and adds the reducer to the search engine.
```

Comment:
```suggestion
 * Loads the automatic facet actions and adds the reducer to the search engine.
```
(if you're curious, this is because `facet` is here operating as an adjective for `actions`, and is therefore not conjugated)